---
title: "Avoiding false discoveries in single-cell RNA-seq by revisiting the first Alzheimers disease dataset"
collection: publications
permalink: /publication/2023-12-04-reanalysis_AD_scRNAseq
excerpt: 'Reanalysis reveals the impact of quality control and differential analysis methods on the discovery of disease-associated genes on the first Alzheimers disease single nucleus RNA-seq dataset.'
date: 2023-12-04
venue: 'eLife'
paperurl: 'https://elifesciences.org/articles/90214'
citation: '<b>Murphy, A. E.</b>, Fancy, N. & Skene, N. G. <i>eLife</i> 12:RP90214 (2023)'
---

Mathys et al. conducted the first single-nucleus RNA-seq (snRNA-seq) study of Alzheimerâ€™s disease (AD) (Mathys et al., 2019). With bulk RNA-seq, changes in gene expression across cell types can be lost, potentially masking the differentially expressed genes (DEGs) across different cell types. Through the use of single-cell techniques, the authors benefitted from increased resolution with the potential to uncover cell type-specific DEGs in AD for the first time. However, there were limitations in both their data processing and quality control and their differential expression analysis. Here, we correct these issues and use best-practice approaches to snRNA-seq differential expression, resulting in 549 times fewer DEGs at a false discovery rate of 0.05. Thus, this study highlights the impact of quality control and differential analysis methods on the discovery of disease-associated genes and aims to refocus the AD research field away from spuriously identified genes.


[Data and Code here](https://github.com/neurogenomics/reanalysis_Mathys_2019)

Recommended citation: <b>Murphy, A. E.</b>, Fancy, N. & Skene, N. G. <i>eLife</i> 12:RP90214 (2023).