---
title: "Predicting gene expression from histone marks using chromatin deep learning models depends on histone mark function, regulatory distance and cellular states"
collection: publications
permalink: /publication/2024-12-11-ChromExpress
excerpt: 'No individual histone mark is consistently the strongest predictor of gene expression across all genomic and cellular contexts. This highlights the need to consider all three factors when determining the effect of histone mark activity on transcriptional state. Furthermore, we conducted in silico histone mark perturbation assays, uncovering functional and disease related loci and highlighting frameworks for the use of chromatin deep learning models to uncover new biological insight.'
date: 2024-12-11
venue: 'Nucleic Acids Research (NAR)'
paperurl: 'https://doi.org/10.1093/nar/gkae1212'
citation: '<b>Murphy, A.E.</b>, Askarova,A. , Lenhard, B. et al. <i>Nucleic Acids Research</i>, gkae1212, (2024).'
---

Understanding how genetic variants affect the epigenome is key to interpreting GWAS, yet profiling these effects across the non-coding genome remains challenging due to experimental scalability. This necessitates accurate computational models. Existing machine learning approaches, while progressively improving, are confined to the cell types they were trained on, limiting their applicability. Here, we introduce Enformer Celltyping, a deep learning model which incorporates distal effects of DNA interactions, up to 100,000 base-pairs away, to predict epigenetic signals in previously unseen cell types. Using DNA and chromatin accessibility data for epigenetic imputation, Enformer Celltyping outperforms current best-in-class approaches and generalises across cell types and biological regions. Moreover, we propose a framework for evaluating models on genetic variant effect prediction using regulatory quantitative trait loci mapping studies, highlighting current limitations in genomic deep learning models. Despite this, Enformer Celltyping can also be used to study cell type-specific genetic enrichment of complex traits.


[Download paper here](http://Al-Murphy.github.io/files/ChromExpress.pdf)

Recommended citation: <b>Murphy, A.E.</b>, Askarova,A. , Lenhard, B. et al. <i>Nucleic Acids Research</i>, gkae1212, (2024).
